src/lib/apis/auths/index.ts

```diff
@@ -110,6 +110,150 @@ export const getSessionUser = async (token: string) => {
 	return res;
 };
 
+export const ldapUserSignIn = async (user: string, password: string) => {
+	let error = null;
+
+	const res = await fetch(`${WEBUI_API_BASE_URL}/auths/ldap`, {
```

Comment:
Use `joined_url = urllib.parse.urljoin(base_url, url_to_join)` to join the Base URL with the ldap path.

Reply:
This is TS and the method you mentioned is for Python. Plus, this is how other functions join the URL parts and I was just following the existing style.